### PR TITLE
Various improvements and corrections for voluntary forms

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
See individual commits for details.

### CODE REVIEW

File: codelists/procurement-procedure-type_procedure-type-14.gc [hunk 1]
```diff
@@ -0,0 +1,765 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--File generated from metadata database-->
+<gc:CodeList xmlns:gc="http://docs.oasis-open.org/codelist/ns/genericode/1.0/">
+   <Identification>
+      <ShortName>ProcedureTypeE5</ShortName>
```
Comment (YvesJo) [inline codelists/procurement-procedure-type_procedure-type-14.gc:5]: may be an issue in the shortname ...

File: codelists/procurement-procedure-type_procedure-type-15.gc [hunk 2]
```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--File generated from metadata database-->
+<gc:CodeList xmlns:gc="http://docs.oasis-open.org/codelist/ns/genericode/1.0/">
+   <Identification>
+      <ShortName>ProcedureTypeE5</ShortName>
```
Comment (YvesJo) [inline codelists/procurement-procedure-type_procedure-type-15.gc:5]: here too

File: codelists/procurement-procedure-type_procedure-type-19.gc [hunk 3]
```diff
@@ -0,0 +1,765 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--File generated from metadata database-->
+<gc:CodeList xmlns:gc="http://docs.oasis-open.org/codelist/ns/genericode/1.0/">
+   <Identification>
+      <ShortName>ProcedureTypeE5</ShortName>
```
Comment (YvesJo) [inline codelists/procurement-procedure-type_procedure-type-19.gc:5]: same here

File: codelists/procurement-procedure-type_procedure-type-32.gc [hunk 4]
```diff
@@ -0,0 +1,765 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--File generated from metadata database-->
+<gc:CodeList xmlns:gc="http://docs.oasis-open.org/codelist/ns/genericode/1.0/">
+   <Identification>
+      <ShortName>ProcedureTypeE5</ShortName>
```
Comment (YvesJo) [inline codelists/procurement-procedure-type_procedure-type-32.gc:5]: idem

File: codelists/procurement-procedure-type_procedure-type-35.gc [hunk 5]
```diff
@@ -0,0 +1,765 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--File generated from metadata database-->
+<gc:CodeList xmlns:gc="http://docs.oasis-open.org/codelist/ns/genericode/1.0/">
+   <Identification>
+      <ShortName>ProcedureTypeE5</ShortName>
```
Comment (YvesJo) [inline codelists/procurement-procedure-type_procedure-type-35.gc:5]: ibidem

**Review by YvesJo — APPROVED** [review]
Didn't detect anything besides the List Shortnames that will get fixed (done with TEDEFO-3672). Shouldn't have any impact however as I do not see these being used

**Comment by bertrand-lorentz** [comment]
I've just re-exported the codelists, and I see that the values of ShortName have been corrected (see added commit).
So I'm merging this.

**Pull request merged**